eZ/Publish/Core/Search/Solr/Content/Gateway/EndpointProvider.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace eZ\Publish\Core\Search\Solr\Content\Gateway;
+
+use eZ\Publish\SPI\Search\FieldType;
+
+/**
+ *
```

Comment:
Added in https://github.com/ezsystems/ezpublish-kernel/commit/7a3e17cd0211a48bc65a9b328e2d2e6b951a2e78